config/serverless.chat.yml

```
@@ -7,7 +7,17 @@ xpack.serverless.chat.enabled: true
 xpack.cloud.serverless.project_type: search
```

Shouldn't we change this to `chat`?

```suggestion
xpack.cloud.serverless.project_type: chat
```